tests/test_crypto.py

```diff
@@ -2809,39 +2965,33 @@ def test_dump_certificate_bad_type(self):
         `dump_certificate` raises a `ValueError` if it's called with
         a bad type.
         """
-        cert = load_certificate(FILETYPE_PEM, cleartextCertificatePEM)
+        cert = load_certificate(FILETYPE_PEM, root_cert_pem)
         with pytest.raises(ValueError):
             dump_certificate(object(), cert)
 
     def test_dump_privatekey_pem(self):
         """
         `dump_privatekey` writes a PEM
         """
-        key = load_privatekey(FILETYPE_PEM, cleartextPrivateKeyPEM)
+        key = load_privatekey(FILETYPE_PEM, root_key_pem)
         assert key.check()
         dumped_pem = dump_privatekey(FILETYPE_PEM, key)
-        assert dumped_pem == cleartextPrivateKeyPEM
+        assert dumped_pem == normalized_root_key_pem
 
     def test_dump_privatekey_asn1(self):
         """
         `dump_privatekey` writes a DER
         """
-        key = load_privatekey(FILETYPE_PEM, cleartextPrivateKeyPEM)
-        dumped_pem = dump_privatekey(FILETYPE_PEM, key)
+        key = load_privatekey(FILETYPE_PEM, root_key_pem)
 
         dumped_der = dump_privatekey(FILETYPE_ASN1, key)
-        # XXX This OpenSSL call writes "writing RSA key" to standard out.  Sad.
-        good_der = _runopenssl(dumped_pem, b"rsa", b"-outform", b"DER")
-        assert dumped_der == good_der
-        key2 = load_privatekey(FILETYPE_ASN1, dumped_der)
```

Comment:
This was the only call to `load_privatekey` with `FILETYPE_ASN1`  in the tests. So now it's uncovered. lol

Reply:
These tests are so bad.